src/docs/integration/linux/build-tools/yocto.md

```diff
@@ -0,0 +1,164 @@
+# Yocto
+
+Peridio provides a yocto layer `meta-peridio` that can be used to include peridiod and its dependencies into your system.
+
+[More information about the meta-peridio layer](https://github.com/peridio/meta-peridio)
```

Comment:
I'd rework to:

The source code for the meta-peridio layer is available on [GitHub](https://github.com/peridio/meta-peridio).